guides/micronaut-security-keys-jwks/micronaut-security-keys-jwks.adoc

```diff
@@ -137,14 +137,17 @@ Invoke the `/keys` endpoint:
 [source, bash]
 ----
 curl -i localhost:8080/keys
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks